settings.php

```diff
@@ -35,5 +35,78 @@
         0
     ));
 
-    // TODO: Options to configure default instance options.
+    // This section configures which contact methods should be displayed.
+    $settings->add(new admin_setting_heading('block_course_contacts/defaultheading',
+            get_string('method', 'block_course_contacts'), ''));
+
+    $yesno = [
+        0 => get_string('no'),
+        1 => get_string('yes')
```

Comment:
I don't think you need to key the numerical values like so. Is this some standard I'm not aware of?